org-formation/650-identity-providers/_tasks.yaml

```diff
@@ -259,7 +259,7 @@ GithubOidcCfnTemplateDeploy:
       - name: "aws-infra"
         branches: ["master"]
       - name: "service-catalog-library"
-        branches: ["master"]
+        branches: [ "master", "v1.1.45" ]
```

Comment:
Do you really want to give the specific branch ( "v1.1.45")? What happens when you go to the next version, e.g.,  "v1.1.46"?